sql/expression/function/json/json_keys_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package json
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gopkg.in/src-d/go-errors.v1"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/types"
+)
+
+func TestJSONKeys(t *testing.T) {
+	_, err := NewJSONKeys()
+	require.True(t, errors.Is(err, sql.ErrInvalidArgumentNumber))
+
+	f1 := buildGetFieldExpressions(t, NewJSONKeys, 1)
+	f2 := buildGetFieldExpressions(t, NewJSONKeys, 2)
+
+	testCases := []struct {
+		f   sql.Expression
+		row sql.Row
+		exp interface{}
+		err bool
+	}{
+		{
+			f:   f1,
+			row: sql.Row{`null`},
+			exp: nil,
+		},
+		{
+			f:   f1,
+			row: sql.Row{`1`},
```

Comment:
In addition to the string `"1"`, it might be nice to also test the number `1`, which looks like it should return an error:
```
select JSON_KEYS(1);
ERROR 3146 (22032): Invalid data type for JSON data in argument 1 to function json_keys; a JSON string or JSON type is required.
```